akka-cluster-tools/src/main/scala/akka/cluster/singleton/ClusterSingletonManager.scala

```diff
@@ -683,7 +731,80 @@ class ClusterSingletonManager(
       self ! DelayedMemberRemoved(m)
   }
 
-  def gotoOldest(): State = {
+  // Try and go to oldest, taking the lease if needed
+  def tryGoToOldest(): State = {
+    // check if lease
+    lease match {
+      case None ⇒
+        goToOldest()
+      case Some(_) ⇒
+        logInfo("Trying to acquire lease before starting singleton")
+        tryAcquireLease()
+    }
+  }
+
+  when(AcquiringLease) {
+    case Event(AcquireLeaseResult(result), _) ⇒
+      logInfo("Acquire lease result {}", result)
```

Comment:
`result` is a boolean, so this will be `Aquire lease result false`, what about individual log entries in the if branches instead with "Acquired lease, becoming oldest" and "Failed to acquire lease, retrying in 5 seconds"

Reply:
:+1: